scipy/sparse/linalg/_isolve/iterative.py

```diff
@@ -169,9 +143,8 @@ def bicg(A, b, x0=None, *, tol=_NoValue, maxiter=None, M=None, callback=None,
         return postprocess(x), maxiter
 
 
-@_deprecate_positional_args(version="1.14")
-def bicgstab(A, b, *, x0=None, tol=_NoValue, maxiter=None, M=None,
-             callback=None, atol=0., rtol=1e-5):
+def bicgstab(A, b, x0=None, *, rtol=1e-5, atol=0., maxiter=None, M=None,
+             callback=None):
```

Comment:
Note that `x0` was falsely marked as kwarg-only here; it was missed by 6476f8aecc75a250164ddc68d9969bac838afb96